cypress/e2e/models/migration/migration-waves/migration-wave.ts

```diff
@@ -151,7 +151,7 @@ export class MigrationWave {
             return;
         }
 
-        cy.get(MigrationWaveView.applicationsSubmitButton).click();
+        cy.get(submitButton).click();
```

Comment:
Here you can also use it I guess

Reply:
using the function submitForm in this case will cause the test to fail...